doc/client-proxy.md

```diff
@@ -75,3 +75,5 @@ Currently, value of `X-Forwarded-Prefix` affects following URLs:
   * `url` key
```

Comment:
Mention that WMS/WFS endpoints do not respect headers
```
        'X-Forwarded-Proto',
        'X-Forwarded-Host',
        'X-Forwarded-For',
        'X-Forwarded-Path',
        'Forwarded',
        'Host',
```
according to GeoServer's documentation.

Reference it also to/from endpoints.md